### PR TITLE
put logs at beggining of log window

### PR DESCRIPTION
**Pull request by Reaver01**

### CODE REVIEW

**Comment by Reaver01** [comment]
:+1: 

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoWeb/pull-request/77/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoWeb)

File: js/main.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 $(document).ready(function() {
   mapView.init();
-  var socket = io.connect('http://' + document.domain + ':' + location.port + '/event');
+  var socket = io.connect('http://' + document.domain + ':8000/event');
```
Comment (MaxLeiter) [inline js/main.js:5]: We shouldn't require a port

**Comment by meyer9** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoWeb/pull-request/77/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoWeb)

**Comment by RedSparr0w** [comment]
im still getting

Exp: null
Exp to Lvl xx: NaN
Kilometers Walked: NaN

But i think this is due to the bot side of things?
As its only happened since using [latest commit on master](https://github.com/OpenPoGo/OpenPoGoBot/tree/0fa223b6143d1d82790b0d1e511fad5928749bc7), i was a few commits behind before that though

**Comment by meyer9** [comment]
yes, working on fixing.

**Comment by RedSparr0w** [comment]
I Think https://github.com/OpenPoGo/OpenPoGoBot/pull/94 fixed my contributions in this PR, as data now seems to be numbers rather than strings coming in.

Also km walked seems to be back in latest commit
but xp still null/NaN

**Comment by BobbyWibowo** [comment]
**EDIT:** Never mind, the catchable handler was broken, or probably wasn't fully compatible with PokemonGoF? Either way, fixed it on my personal build.

File: js/main.js (hunk 1 above)
Comment (aslafy-z) [inline js/main.js:5]: Server port should be configurable withing userData.js

**Comment by MaxLeiter** [comment]
status on this?

**Pull request closed without merging**